### DISCUSSION TITLE
Welcome to Urbanity Network Data Page!

### DISCUSSION THREAD

**Discussion by winstonyym** [winstonyym/urbanity#1] [Announcements]
## 👋 Welcome!
  This page lists the set of cities currently available in the Urbanity Network Dataset (sorted by alphabetical order):
  * Adelaide
  * Amsterdam
  * Antwerp
  * Athens
  * Atlanta
  * Auckland
  * Austin
  * Barcelona
  * Belo Horizonte
  * Berlin
  * Bern
  * Bangkok
  * Bogota
  * Boston
  * Brisbane
  * Budapest
  * Buenos Aires
  * Campinas
  * Chiang Mai
  * Chicago
  * Denver
  * Edinburgh
  * Glasgow
  * Hanoi
  * Houston
  * Johannesburg
  * Kowloon
  * Kuala Lumpur
  * Madrid
  * Manila
  * Melbourne
  * Mexico
  * Miami
  * Milan
  * Paris
  * Phoenix
  * San Jose
  * Santiago
  * Sao Paulo
  * Sapporo
  * Seattle
  * Singapore
  * Sydney
  * Taichung
  * Taipei
  * Tokyo
  * Washington D.C
  * Yokohama
  * Zagreb
  * Zurich

 If your city of interest is not included, please comment below and we will add it into the next phase as a priority. Note: Urbanity depends on availability of crowdsourced VGI data. In this regard, one of the main limitations towards expanding Urbanity to more cities is the availability of crowdsourced street view imagery. On this aspect, users should first proceed to [Mapillary](https://www.mapillary.com/app/) or [KartaView](https://kartaview.org/landing) to check if street view imagery is available for your city of interest.

**Comment by gerardoezequiel** [comment]
Greater London would be an amazing contribution!

**Comment by datsvarun** [comment]
Great Mumbai, Navi Mumbai, Delhi, Bengaluru, Pune would be some essential additions, being some of the most rapidly urbanizing cities out there.

**Comment by haugekugga14** [comment]
Hong Kong, Ghent
